ci/do_ci.sh

```diff
@@ -19,7 +19,6 @@ cp -f "$UPSTREAM_ENVOY_SRCDIR/.bazelrc"                    "$SOURCE_DIR/upstream
 cp -f "$UPSTREAM_ENVOY_SRCDIR/.bazelversion"               "$SOURCE_DIR/.bazelversion"
 cp -f "$UPSTREAM_ENVOY_SRCDIR/ci/WORKSPACE.filter.example" "$SOURCE_DIR/ci/"
 cp -f "$UPSTREAM_ENVOY_SRCDIR/VERSION.txt"                 "$SOURCE_DIR/VERSION.txt"
-cp -f "$UPSTREAM_ENVOY_SRCDIR/tools/shell_utils.sh"        "$SOURCE_DIR/tools"
```

Comment:
Are we making this change because don't use this file? Or does it no longer exist at `$UPSTREAM_ENVOY_SRCDIR/tools/shell_utils.sh`?

Reply:
The file no longer exists -- it was deleted upstream with https://github.com/envoyproxy/envoy/commit/07bf8f427386f4c95d658608f334f1f8c5a38a1c#diff-0671c871cea874cab5703e16d583fbb867d618fa82a92d0080fe07c92a692669

It looks like it used to contain python venv tooling that is no longer used/needed